processing/src/main/java/io/druid/query/groupby/epinephelinae/GroupByQueryEngineV2.java

```diff
@@ -159,57 +160,29 @@ public GroupByEngineIterator make()
                             final ByteBuffer buffer = bufferHolder.get();
```

Comment:
Hmm I tried, but not sure how I can further improve without changing the sequence order.